src/main/java/org/skriptlang/skript/bukkit/recipes/elements/ExprGetRecipe.java

```diff
@@ -0,0 +1,73 @@
+package org.skriptlang.skript.bukkit.recipes.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.bukkitutil.NamespacedUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.Bukkit;
+import org.bukkit.NamespacedKey;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.Recipe;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Recipe")
+@Description("Returns the recipe registered with the provided name.")
+@Examples({
+	"set {_recipe} to recipe with the key \"my_recipe\"",
+	"set {_recipes::*} to recipes with the ids \"my_recipe\" and \"custom_recipe\""
+})
+@Since("INSERT VERSION")
+public class ExprGetRecipe extends SimpleExpression<Recipe> {
+
+	static {
+		Skript.registerExpression(ExprGetRecipe.class, Recipe.class, ExpressionType.SIMPLE,
+			"[the] recipe[s] [with [the] (key|id)[s]] %strings%");
```

Comment:
Personally not a big fan of this. Simply because it wouldnt match the behavior, also it could imply that you can set the name of it. That's just my opinion though

Reply:
agreed